manubulon requires exec permission on satellites & cpu/mem thres

moix opened the pull request:
manubulon requires exec permission on satellites & cpu/mem threshold for check_snmp_process

related to #5129 #5128

dnsmichi commented on itl/command-plugins-manubulon.conf:
```diff
@@ -149,15 +149,15 @@ object CheckCommand "snmp-storage" {
 	import "snmp-manubulon-command"
 
-	command = [ ManubulonPluginDir + "/check_snmp_int.pl" ]
+	command = [ "perl", ManubulonPluginDir + "/check_snmp_int.pl" ]
```
As explained in #5128 we won't add such a workaround for syncing scripts via **config sync**.

dnsmichi commented on itl/command-plugins-manubulon.conf:
```diff
@@ -244,12 +244,23 @@ object CheckCommand "snmp-process" {
+	vars.snmp_process_mem_usage = false
+	vars.snmp_process_mem_thres = "0,0"
+	vars.snmp_process_cpu_usage = false
+	vars.snmp_process_cpu_thres = "0,0"
```
Can you please rename `thres` to `threshold` and add the `description` field with additional details? Furthermore please update the documentation in doc/10-icinga-template-library.md.

moix commented:
Hi @dnsmichi , sure I changed as suggested:
* renamed thres to threshold
* added description
* updated documentation in doc/10-icinga-template-library.md accordingly

Thanks
